File: Lib/Telerik/Examples/QuickStart/TreeView/TreeExampleHeaderPanel.vb
Imports System.Text
Imports Telerik.WinControls.UI

Namespace Telerik.Examples.WinControls.TreeView
	Public Class TreeExampleHeaderPanel
		Inherits RadPanel
		Protected Overrides Sub OnLoad(ByVal desiredSize As Size)
            MyBase.OnLoad(desiredSize)

			Me.PanelElement.Font = New Font("Segoe UI", 10, FontStyle.Bold)
			Me.PanelElement.Padding = New Padding(8, 4, 2, 2)
		End Sub

        Public Overrides Property ThemeClassName() As String
            Get
                Return GetType(RadPanel).FullName
            End Get

            Set(value As String)

            End Set
        End Property

	End Class
End Namespace

File: Lib/Telerik/Examples/QuickStart/TreeView/TreeExampleHeaderPanel.cs
using System.Drawing;
using Telerik.WinControls.UI;

namespace Telerik.Examples.WinControls.TreeView
{
    public class TreeExampleHeaderPanel: RadPanel
    {
        protected override void OnLoad(Size desiredSize)
        {
            base.OnLoad(desiredSize);

            this.PanelElement.Font = new Font("Segoe UI", 10, FontStyle.Bold);
            this.PanelElement.Padding = new System.Windows.Forms.Padding(8, 4, 2, 2);
        }

        public override string ThemeClassName
        {
            get
            {
                return typeof(RadPanel).FullName;
            }
        }
    }
}
